dev/devicelab/lib/tasks/analysis.dart

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:meta/meta.dart';
+import 'package:path/path.dart' as path;
+
+import '../framework/benchmarks.dart';
+import '../framework/framework.dart';
+import '../framework/utils.dart';
+
+TaskFunction createAnalyzerCliTest(
+    {@required String sdk,
```

Comment:
put the { on the line with the ( so that the whole thing lines up better
similarly the `}) {` can go on a line of its own so the whole thing looks better